components/payment_form/gather_intent/gather_intent.scss

```diff
@@ -0,0 +1,171 @@
+        font-weight: 600 !important;
+        line-height: 24px !important;
+    }
+
+    &__button {
+        height: auto !important;
+        padding: 0 !important;
+        border: none !important;
+        background: transparent !important;
+        color: var(--button-bg) !important;
+        font-size: 14px !important;
+        font-weight: 400 !important;
+        line-height: 20px !important;
+    }
+}
+
+.savedFeedback__text {
+    align-self: center;
+    color: rgba(63, 67, 80, 1);
+    font-size: 22px;
+    font-weight: 600;
+}
+
+.GatherIntentModal {
+    .modal-content {
+        max-width: 512px;
+        max-height: 465px;
+        border-radius: 12px;
+    }
+
+    .modal-dialog {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        min-width: 532px;
+        margin: auto;
+        transform: translate(-50%, -50%) !important;
+    }
+
+    .modal-footer {
+        padding-top: 24px;
+        padding-bottom: 24px;
+        border-radius: 0 0 12px 12px;
+    }
+
+    .modal-header {
+        border-radius: 12px 12px 0 0;
+    }
+
+    .modal-header,
+    .modal-footer {
+        display: flex;
+        padding-right: 32px;
+        padding-left: 32px;
+    }
+
+    .GatherIntentModal__header {
+        align-items: baseline;
+        border: none;
+
+        &.modal-header {
+            background: var(--center-channel-bg);
+        }
+
+        h3 {
+            margin: 0;
+            color: rgba(63, 67, 80, 1);
+            font-size: 22px;
+            font-weight: bold;
+            line-height: 28px;
+        }
+
+        .icon-close {
+            padding: 0;
+            border: none;
+            margin-left: auto;
+            background: var(--center-channel-bg);
+            color: rgba(63, 67, 80, 1);
+            font-size: 24px;
+        }
+    }
+
+    .modal-body {
+        display: block;
+        overflow: auto;
+        padding: 0 32px;
+        margin-bottom: 28px;
+        color: rgba(63, 67, 80, 1);
+    }
+
+    &__submitted-icon-container {
+        margin-bottom: 24px;
+
+        > svg {
+            width: 51px;
+            height: 51px;
+            align-self: center;
+            color: rgba(61, 184, 135, 1);
+        }
+    }
+
+    &__body {
+        text-align: center;
+
+        &__question {
+            margin-bottom: 10px;
+            color: rgba(63, 67, 80, 1);
+            font-size: 14px;
+            font-weight: 600;
+        }
+
+        &__option {
+            margin-top: 0;
+            margin-bottom: 10px;
+            color: rgba(63, 67, 80, 1);
+            font-size: 14px;
+            font-weight: 400;
+        }
+
+        &__label {
+            padding-left: 12px;
+            cursor: default;
+            vertical-align: bottom;
+        }
+
+        &__checkbox {
+            height: 28px;
+            vertical-align: middle;
+        }
+
+        &__textarea {
+            width: 100%;
+            border: solid 1px rgba(63, 67, 80, 0.16);
```

Comment:
I think we'll also want to explicitly set the background for the textarea to `var(--center-channel-bg)` so that in dark mode it looks like the inputs on the form that opens this modal:
![dark-theme-pre](https://user-images.githubusercontent.com/13738432/195893218-8bebb919-2e7a-483c-8db5-9d5ad273c20e.png)

Success modal looks good to me in dark mode.

Reply:
Another test case to add to my list, thanks!.

Reply:
The font color on the dark mode theme still seems off. Can use `rgba(var(--center-channel-color-rgb), 0.72)`
<img width="1792" alt="Screenshot 2022-10-19 at 19 17 11" src="https://user-images.githubusercontent.com/28563179/196747842-eb68c09f-b373-4f73-91fc-5e127b132fb3.png">